### ISSUE TITLE
Even though I have "do not transfer" checked for certain pokemon, it still transfers??

### ISSUE THREAD

**Issue by FlyingFish10** [finndev/PokeBuddy#493]
I don't want it to transfer a dragonite if I get another one just because it's worse than my first one, how do I stop this???

**Comment by IamDeric** [comment]
Make them favorite manually

**Comment by Rab89** [comment]
yep @IamDeric is right you have to favourite the specific pokemon on the smatphone app

**Comment by FlyingFish10** [comment]
but if I open the app it'll teleport me, causing a ban

**Comment by curtiola** [comment]
Then use quickspin afterwards or something

**Comment by Rab89** [comment]
nope aslong as you dont loot any pokestops or catch any pokemon, the only things i know of that you can do is set egg incubation and favourite and transfer/evolve pokemon.

but be sure to stop the bot before using the smartphone app though

**Comment by psblackz** [comment]
if i fav transfer in the same way
